content/en-us/reference/engine/globals/LuaGlobals.yaml

```diff
@@ -125,6 +125,12 @@ functions:
         current script. When using `Global.LuaGlobals.getfenv()` to get the
         current environment of a script, it will return the same table every
         time within the specific thread.
+
+      **WARNING:** This function allows uncontrolled change of the global/function
```

Comment:
Adding this up here; it's shown in the `deprecation_message:` field, but those are currently not rendering as expected.

Reply:
I'm pretty sure loadstring should be deprecated anyway because you need a setting to activate it but thats not for here :+1: